### PR TITLE
release: Amplify Android 1.17.3

### PR DESCRIPTION
**Pull request by awsmobilesdk**
## Release PR review checklist
- [x] Verify version name in gradle.properties
- [x] Verify CHANGELOG.md

## Module versions

Amplify Android 1.17.3\nAmplify Android 0.1.4

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- **datastore:** fix unit tests by adding createdAt updatedAt (#1266)
+- manually triggered workflow to start release process (#1196)
+- **build:** add buildspec for publishing to maven (#1271)
+- small updates to the create_next_release_pr workflow (#1278)
+- update release number in the README.md file (#1277)
+- update name of workflow that prepares next release (#1280)
+- bump sdk version to 2.22.6 (#1281)
+- use shared lane file (#1282)
+
+[See all changes between 1.17.2 and 1.17.3](https://github.com//compare/release_v1.17.2...release_v1.17.3)
```
Comment (richardmcclellan) [inline CHANGELOG.md:20]:
```suggestion
[See all changes between 1.17.2 and 1.17.3](https://github.com/aws-amplify/amplify-android/compare/release_v1.17.2...release_v1.17.3)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+### 
+- **datastore:** fix unit tests by adding createdAt updatedAt (#1266)
+- manually triggered workflow to start release process (#1196)
+- **build:** add buildspec for publishing to maven (#1271)
+- small updates to the create_next_release_pr workflow (#1278)
+- update release number in the README.md file (#1277)
+- update name of workflow that prepares next release (#1280)
+- bump sdk version to 2.22.6 (#1281)
+- use shared lane file (#1282)
```
Comment (richardmcclellan) [inline CHANGELOG.md:18]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Release 1.17.3
+
+### Bug Fixes
+- **datastore:** optimize sync queries with predicates by wrapping in an AND group (#1225)
+- **kotlin:** getCurrentUser return type should be nullable (#1265)
+- **ci:** update branch name and PR subject (#1276)
+- change protocol for github import (#1284)
```
Comment (richardmcclellan) [inline CHANGELOG.md:7]:
```suggestion
```

File: core-kotlin/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Release 0.1.4
+
+### Bug Fixes
+- **datastore:** optimize sync queries with predicates by wrapping in an AND group (#1225)
+- **kotlin:** getCurrentUser return type should be nullable (#1265)
+- **ci:** update branch name and PR subject (#1276)
+- change protocol for github import (#1284)
```
Comment (richardmcclellan) [inline core-kotlin/CHANGELOG.md:7]:
```suggestion
```

File: core-kotlin/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+### 
+- **datastore:** fix unit tests by adding createdAt updatedAt (#1266)
+- manually triggered workflow to start release process (#1196)
+- **build:** add buildspec for publishing to maven (#1271)
+- small updates to the create_next_release_pr workflow (#1278)
+- update release number in the README.md file (#1277)
+- update name of workflow that prepares next release (#1280)
+- bump sdk version to 2.22.6 (#1281)
+- use shared lane file (#1282)
```
Comment (richardmcclellan) [inline core-kotlin/CHANGELOG.md:18]:
```suggestion
```

File: core-kotlin/CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+[See all changes between 0.1.3 and 0.1.4](https://github.com//compare/release-kotlin_v0.1.3...release-kotlin_v0.1.4)
```
Comment (richardmcclellan) [inline core-kotlin/CHANGELOG.md:20]:
```suggestion
[See all changes between 0.1.3 and 0.1.4](https://github.com/aws-amplify/amplify-android/compare/release-kotlin_v0.1.3...release-kotlin_v0.1.4)
```

**Review by richardmcclellan — APPROVED** [review]
Looks like we need to tweak the `CHANGELOG` generation a bit, but I'm fine with manually fixing it up for now, if you want.   Would be great to get a release out today if we can!

**Review by rjuliano — APPROVED** [review]

**Pull request merged**